validator/sawtooth_validator/journal/publisher.py

```diff
@@ -32,10 +32,185 @@
 from sawtooth_validator.protobuf.block_pb2 import BlockHeader
 from sawtooth_validator.protobuf.transaction_pb2 import TransactionHeader
 
+from sawtooth_validator.state.config_view import ConfigView
 
 LOGGER = logging.getLogger(__name__)
 
 
+class CandidateBlock(object):
```

Comment:
Small nit....and all my comments probably are going to be....if this is meant to only be used within `publisher.py`, do you want to preprend `_` to name to indicate "private"?